src/App.js

```diff
@@ -4,13 +4,23 @@ import { Fragment } from 'react';
 import SignUpForm from './components/SignupForm';
 import PrivateRoute from './PrivateRoute';
 import Dashboard from './components/dashboard/Dashboard';
-import { Route } from 'react-router-dom';
+import LandingPage from './components/LandingPage';
+import LoginForm from './components/LoginForm';
+import TaskPage from './components/TaskPage';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 const App = () => {
   return (
     <Fragment>
-      <Route exact path="/signup" component={SignUpForm} />
-      <PrivateRoute exact path="/dashboard" component={Dashboard} />
+      <Router>
```

Comment:
The `development` branch has `App` wrapped in the `Router` up a level in the `index.js` file, so you don't need to wrap it in a `Router` again here